cumulusci/core/keychain/tests/test_encrypted_file_project_keychain.py

```diff
@@ -92,6 +133,10 @@ def test_remove_org__global_not_found(self, keychain):
         with pytest.raises(OrgNotFound):
             keychain.remove_org("test", True)
 
+    @pytest.mark.skipif(
+        sys.platform.startswith("win"),
+        reason="os.stat() returns a differing value on windows",
+    )
```

Comment:
Can we skip the specific assertion about file mode (or make what it asserts conditional based on platform) rather than skipping the entire test?